.github/workflows/build_and_test.yml

```diff
@@ -17,7 +17,7 @@ permissions:
 env:
   GCC_VERSION: "11"
   LLVM_VERSION: "17"
-  COMMON_CMAKE_FLAGS: |
+  COMMON_CMAKE_FLAGS: >
```

Comment:
Just out of curiosity what does it change?

Reply:
It's YAML syntax, it allows the following multi-line string to be single line. See https://yaml-multiline.info/